SingularityUI/app/components/requestForm/RequestForm.jsx

```diff
@@ -295,7 +298,7 @@ class RequestForm extends React.Component {
     const rackOptions = _.pluck(this.props.racks, 'id').map(id => ({value: id, label: id}));
     const rackAffinity = (
       <div className="form-group">
-        <label htmlFor="rack-affinity">Rack Affinity <span className="form-label-tip">separate multiple racks with commas</span></label>
+        <label htmlFor="rack-affinity">Rack Affinity <span className="form-label-tip">choose any sub</span></label>
```

Comment:
Yeah since the new component lets you select multiple without the need for commas. I made a typo though, should be sub => subset